src/main/java/nextstep/courses/infrastructure/JdbcCoverImageRepository.java

```diff
@@ -0,0 +1,56 @@
+package nextstep.courses.infrastructure;
+
+import nextstep.courses.domain.image.CoverImage;
+import nextstep.courses.domain.image.CoverImageExtension;
+import nextstep.courses.domain.image.CoverImageFileSize;
+import nextstep.courses.domain.image.CoverImagePixelSize;
+import nextstep.courses.domain.repository.CoverImageRepository;
+import org.springframework.jdbc.core.JdbcOperations;
+import org.springframework.stereotype.Repository;
+
+import java.util.Optional;
+
+@Repository("coverImageRepository")
```

Comment:
```suggestion
@Repository
```
현재는 구현체가 하나이기 때문에 따로 명시해주지 않아도 좋을 것 같아요 😄